src/national.py

```diff
@@ -204,8 +214,11 @@ def get_national_pvlive(
     """
     logger.info(f"Get national PV Live estimates values " f"for regime {regime} for  {user}")
 
+    if is_fake:
+        make_fake_gsp_yields(gsp_ids=List[gsp_id], session=session)
```

Comment:
should this be `make_fake_gsp_yields(gsp_ids=[gsp_id], session=session)`

Reply:
Awesome! Thanks for this. Looks like it's worked!